### PR TITLE
oversecured 1.0.1

### PR DESCRIPTION
**Pull request by bagipro**
TagCheck

### What to do if the build fails?

At the moment contributors do not have access to the CI workflow triggered by StepLib PRs. In case of a failed build, we ask for your patience. Maintainers of Bitrise Steplib will sort it out for you or inform you if any further action is needed.

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [+] __I will not move an already shared step version's tag to another commit__
- [+] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [+] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [+] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [+] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

File: steps/oversecured/1.0.1/step.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- android
+type_tags:
+- test
+toolkit:
+  go:
+    package_name: github.com/oversecured/oversecured-bitrise-step
+is_requires_admin_user: true
```
Comment (tadam313) [inline steps/oversecured/1.0.1/step.yml:23]: can you please remove this? It is deprecated now and has no effect :)

**Review by tadam313 — CHANGES_REQUESTED** [review]
thanks for the contribution, I have one tiny request, and please also sign the CLA for this contribution :) thanks 🙏

**Comment by Janos-vrancsik** [comment]
> thanks for the contribution, I have one tiny request, and please also sign the CLA for this contribution :) thanks 🙏

@tadam313, @bagipro already signed the CLA for the initial Oversecured step contribution. AFAIK, he doesn't need to sign it again. :)

**Comment by tadam313** [comment]
> > thanks for the contribution, I have one tiny request, and please also sign the CLA for this contribution :) thanks 🙏
> 
> @tadam313, @bagipro already signed the CLA for the initial Oversecured step contribution. AFAIK, he doesn't need to sign it again. :)

sorry, we checked it and indeed it is signed, our CLA automation did not show it, we'll check the reasons why. This case, please disregard that part :)

**Comment by bagipro** [comment]
Hey,
@tadam313 

sorry for the long response, updated this PR

@Janos-vrancsik 
yup, for some reason it always displays that I didn't accept it

**Review by tadam313 — APPROVED** [review]
LGTM

**Pull request merged**